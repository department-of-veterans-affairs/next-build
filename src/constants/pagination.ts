export const DEFAULT_PAGE_LIST_LENGTH = 8
