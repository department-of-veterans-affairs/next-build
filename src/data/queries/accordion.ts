import { QueryFormatter } from 'next-drupal-query'
import { ParagraphAccordion } from '@/types/drupal/paragraph'
import { Accordion } from '@/types/formatted/accordion'

export const formatter: QueryFormatter<ParagraphAccordion, Accordion> = (
  entity: ParagraphAccordion
) => {
  return {
    id: entity.id,
    header: entity.field_header || null,
    html: entity.field_rich_wysiwyg.processed,
  }
}
