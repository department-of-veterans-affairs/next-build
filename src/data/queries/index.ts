import { createQueries } from 'next-drupal-query'
import * as NewsStory from './newsStory'
import * as NewsStoryTeaser from './newsStoryTeaser'
import * as StoryListing from './storyListing'
import * as QuestionAnswer from './questionAnswer'
import * as ExpandableText from './expandableText'
import * as LinkTeaser from './linkTeaser'
import * as MediaImage from './mediaImage'
import * as Banner from './banners'
import * as PersonProfile from './personProfile'
import * as Button from './button'
import * as AudienceTopics from './audienceTopics'
import * as Alert from './alert'
import * as EmailContact from './emailContact'
import * as BenefitsHub from './benefitsHub'
import * as Wysiwyg from './wysiwyg'
import * as StaticPathResources from './staticPathResources'
import * as HeaderFooter from './headerFooter'

export const queries = createQueries({
  // standard Drupal entity data queries
  'node--news_story': NewsStory,
  'node--news_story--teaser': NewsStoryTeaser,
  'node--story_listing': StoryListing,
  'node--q_a': QuestionAnswer,
  'node--person_profile': PersonProfile,
  'node--landing_page': BenefitsHub, // "Benefits Hub Landing Page"
  'paragraph--audience_topics': AudienceTopics,
  'paragraph--button': Button,
  'paragraph--email_contact': EmailContact,
  'paragraph--expandable_text': ExpandableText,
  'paragraph--link_teaser': LinkTeaser,
  'paragraph--rich_text_char_limit_1000': Wysiwyg,
  'paragraph--wysiwyg': Wysiwyg,
  'media--image': MediaImage,
  'block--alert': Alert,

  // Custom queries 
  'banner--alerts_lookup': Banner,
  'header-footer-data': HeaderFooter,

  // Static Path Generation
  'static-path-resources': StaticPathResources,
})
