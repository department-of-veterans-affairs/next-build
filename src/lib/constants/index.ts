export const HEADER_FOOTER_PATH =
  'https://www.va.gov/generated/headerFooter.json'

export const S3_URL =
  'https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov'

export const DEV_PATH = 'https://dev.cms.va.gov'

// Build Page nodes: params
export const FIELDS =
  'field_answer,field_alert_single.field_alert_non_reusable_ref.field_va_paragraphs,field_buttons,field_tags.field_audience_beneficiares,field_related_information,field_related_benefit_hubs,field_contact_information.field_contact_default'

export const VA_WRAPPER_IMAGES = {
  logo: `${S3_URL}/img/header-logo.png`,
  footer: 'https://www.va.gov/img/homepage/va-logo-white.png',
  usFlag: `${S3_URL}/img/tiny-usa-flag.png`,
  httpsIcon: `${S3_URL}/img/icon-https.svg`,
  arrowIcon: `${S3_URL}/img/arrow-right-white.svg`,
  dotIcon: `${S3_URL}/img/icon-dot-gov.svg`,
  metaImg: `https://www.va.gov/img/design/logo/va-og-image.png`,
}
// content
export const US_FLAG_MSG =
  ' An official website of the United States government'

export const RESOURCE_TYPES = [
  'node--q_a',
  'node--person_profile',
  'paragraph--staff_profile',
]
