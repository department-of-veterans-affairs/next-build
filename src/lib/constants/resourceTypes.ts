export const RESOURCE_TYPES = {
  BENEFITS_DETAIL_PAGE: 'node--page',
  BENEFITS_HUB: 'node--landing_page',
  EVENT_LISTING: 'node--event_listing',
  EVENT: 'node--event',
  LEADERSHIP_LISTING: 'node--leadership_listing',
  LOCATIONS_LISTING: 'node--locations_listing',
  MEDIA: 'media--image',
  PRESS_RELEASE_LISTING: 'node--press_releases_listing',
  PRESS_RELEASE: 'node--press_release',
  QA: 'node--q_a',
  RESOURCES_SUPPORT: 'node--support_resources_detail_page',
  STAFF_PROFILE: 'node--person_profile',
  STORY_LISTING: 'node--story_listing',
  STORY: 'node--news_story',
  SUPPORT_SERVICES: 'node--support_service',
  VAMC_FACILITY: 'node--health_care_local_facility',
  VAMC_SYSTEM_VA_POLICE: 'node--vamc_system_va_police',
  VAMC_SYSTEM: 'node--health_care_region_page',
  VAMC_SYSTEM_REGISTER_FOR_CARE: 'node--vamc_system_register_for_care',
  VAMC_SYSTEM_BILLING_INSURANCE: 'node--vamc_system_billing_insurance',
  VAMC_SYSTEM_MEDICAL_RECORDS_OFFICE: 'node--vamc_system_medical_records_offi',
  VBA_FACILITY: 'node--vba_facility',
  VBA_SERVICE: 'node--vba_facility_service',
  VHA_FACILITY_NONCLINICAL_SERVICE: 'node--vha_facility_nonclinical_service',
  VET_CENTER_HEALTH_SERVICES: 'node--vet_center_facility_health_servi',
  VET_CENTER_OUTSTATION: 'node--vet_center_outstation',
  VET_CENTER: 'node--vet_center',
  VET_CENTER_LOCATION_LISTING: 'node--vet_center_locations_list',
  VET_CENTER_CAP: 'node--vet_center_cap',
  VET_CENTER_MOBILE_VET_CENTER: 'node--vet_center_mobile_vet_center',
  VAMC_HEALTH_SERVICES_LISTING: 'node--health_services_listing',
  VAMC_OPERATING_STATUS_AND_ALERTS: 'node--vamc_operating_status_and_alerts',
  VAMC_SYSTEM_POLICIES_PAGE: 'node--vamc_system_policies_page',
  VAMC_SYSTEM_SERVICE_DES: 'node--regional_health_care_service_des',
  VAMC_SYSTEM_DETAIL_PAGE: 'node--health_care_region_detail_page',
} as const

// (Node) resources that generate full pages.
// Other entries in RESOURCE_TYPES are Drupal
// nodes that are entity references but that
// do not have individual pages.
export const PAGE_RESOURCE_TYPES = [
  RESOURCE_TYPES.BENEFITS_DETAIL_PAGE,
  RESOURCE_TYPES.BENEFITS_HUB,
  RESOURCE_TYPES.EVENT,
  RESOURCE_TYPES.EVENT_LISTING,
  RESOURCE_TYPES.LOCATIONS_LISTING,
  RESOURCE_TYPES.RESOURCES_SUPPORT,
  RESOURCE_TYPES.PRESS_RELEASE,
  RESOURCE_TYPES.PRESS_RELEASE_LISTING,
  RESOURCE_TYPES.STAFF_PROFILE,
  RESOURCE_TYPES.STORY,
  RESOURCE_TYPES.STORY_LISTING,
  RESOURCE_TYPES.VET_CENTER,
  RESOURCE_TYPES.VET_CENTER_OUTSTATION,
  RESOURCE_TYPES.VAMC_FACILITY,
  RESOURCE_TYPES.VAMC_SYSTEM_VA_POLICE,
  RESOURCE_TYPES.VAMC_SYSTEM,
  RESOURCE_TYPES.VAMC_SYSTEM_REGISTER_FOR_CARE,
  RESOURCE_TYPES.VAMC_SYSTEM_BILLING_INSURANCE,
  RESOURCE_TYPES.VAMC_SYSTEM_MEDICAL_RECORDS_OFFICE,
  RESOURCE_TYPES.VBA_FACILITY,
  RESOURCE_TYPES.LEADERSHIP_LISTING,
  RESOURCE_TYPES.VET_CENTER_LOCATION_LISTING,
  RESOURCE_TYPES.VAMC_HEALTH_SERVICES_LISTING,
  RESOURCE_TYPES.VAMC_OPERATING_STATUS_AND_ALERTS,
  RESOURCE_TYPES.VAMC_SYSTEM_POLICIES_PAGE,
  RESOURCE_TYPES.VAMC_SYSTEM_DETAIL_PAGE,
]

export const PARAGRAPH_RESOURCE_TYPES = {
  ACCORDION_ITEM: 'paragraph--basic_accordion',
  ADDRESS: 'paragraph--address',
  ALERT: 'paragraph--alert',
  ALERT_NON_REUSABLE: 'paragraph--non_reusable_alert',
  ALERT_SINGLE: 'paragraph--alert_single',
  AUDIENCE_TOPICS: 'paragraph--audience_topics',
  BUTTON: 'paragraph--button',
  CENTRALIZED_CONTENT_DESCRIPTOR: 'paragraph--centralized_content_descriptor',
  CHECKLIST: 'paragraph--checklist',
  CHECKLIST_ITEM: 'paragraph--checklist_item',
  COLLAPSIBLE_PANEL: 'paragraph--collapsible_panel',
  COLLAPSIBLE_PANEL_ITEM: 'paragraph--collapsible_panel_item',
  CONTACT_INFORMATION: 'paragraph--contact_information',
  DOWNLOADABLE_FILE: 'paragraph--downloadable_file',
  EMAIL_CONTACT: 'paragraph--email_contact',
  EMBEDDED_VIDEO: 'paragraph--embedded_video',
  EXPANDABLE_TEXT: 'paragraph--expandable_text',
  FEATURED_CONTENT: 'paragraph--featured_content',
  HEALTH_CARE_LOCAL_FACILITY_SERVICE:
    'paragraph--health_care_local_facility_servi',
  LINK_TEASER: 'paragraph--link_teaser',
  LINK_TEASER_WITH_IMAGE: 'paragraph--link_teaser_with_image',
  LISTS_OF_LINKS: 'paragraph--lists_of_links',
  LIST_OF_LINKS: 'paragraph--list_of_links',
  LIST_OF_LINK_TEASERS: 'paragraph--list_of_link_teasers',
  MEDIA: 'paragraph--media',
  MEDIA_LIST_IMAGES: 'paragraph--media_list_images',
  MEDIA_LIST_VIDEOS: 'paragraph--media_list_videos',
  NUMBER_CALLOUT: 'paragraph--number_callout',
  PHONE_CONTACT: 'paragraph--phone_number',
  PROCESS_LIST: 'paragraph--process',
  REACT_WIDGET: 'paragraph--react_widget',
  RICH_TEXT_CHAR_LIMIT_1000: 'paragraph--rich_text_char_limit_1000',
  SERVICE_LOCATION: 'paragraph--service_location',
  SERVICE_LOCATION_ADDRESS: 'paragraph--service_location_address',
  SPANISH_TRANSLATION_SUMMARY: 'paragraph--spanish_translation_summary',
  STAFF_PROFILE: 'paragraph--staff_profile',
  STEP: 'paragraph--step',
  STEP_BY_STEP: 'paragraph--step_by_step',
  TABLE: 'paragraph--table',
  WYSIWYG: 'paragraph--wysiwyg',
  QA: 'paragraph--q_a',
  QA_SECTION: 'paragraph--q_a_section',
  QA_GROUP: 'paragraph--q_a_group',
} as const

export const BANNER_RESOURCE_TYPES = {
  BASIC: 'banner',
  PROMO: 'promo_banner',
} as const

export type ResourceType = (typeof RESOURCE_TYPES)[keyof typeof RESOURCE_TYPES]
export type PageResourceType = (typeof PAGE_RESOURCE_TYPES)[number]
export type ParagraphResourceType =
  (typeof PARAGRAPH_RESOURCE_TYPES)[keyof typeof PARAGRAPH_RESOURCE_TYPES]
export type BannerResourceType =
  (typeof BANNER_RESOURCE_TYPES)[keyof typeof BANNER_RESOURCE_TYPES]
