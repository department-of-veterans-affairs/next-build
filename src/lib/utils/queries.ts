import { queries, ParamsType } from '@/lib/drupal/queries'

export const getNestedIncludes = (
  fieldName: string,
  resourceType: ParamsType | ParamsType[]
) => {
  const resourceTypes = Array.isArray(resourceType)
    ? resourceType
    : [resourceType]

  const includedFields = resourceTypes.flatMap(
    (resourceType) =>
      queries.getParams(resourceType).getQueryObject().include?.split(',') ?? []
  )

  return fieldName
    ? [
        fieldName,
        ...includedFields.map(
          (includedField) => `${fieldName}.${includedField}`
        ),
      ]
    : includedFields
}
