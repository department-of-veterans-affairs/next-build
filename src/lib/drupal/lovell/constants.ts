import { RESOURCE_TYPES } from '@/lib/constants/resourceTypes'
import { Administration } from '@/types/formatted/administration'

export const LOVELL = {
  /**
   * The "federal" variant isn't actually one of the variants, but it's used in Drupal to
   * house the bifurcated pages that we wish to define only once and generate copies of
   * for both TRICARE and VA. It is a "Section" (Administration) in Drupal that resources
   * can be assigned to.
   */
  federal: {
    title: 'Lovell Federal health care',
    administration: {
      entityId: 347,
      name: 'Lovell Federal health care',
    } as Administration,
    pathSegment: 'lovell-federal-health-care',
    variant: 'federal',
  },
  tricare: {
    title: 'Lovell Federal health care - TRICARE',
    administration: {
      entityId: 1039,
      name: 'Lovell - TRICARE',
    } as Administration,
    pathSegment: 'lovell-federal-health-care-tricare',
    variant: 'tricare',
  },
  va: {
    title: 'Lovell Federal health care - VA',
    administration: {
      entityId: 1040,
      name: 'Lovell - VA',
    } as Administration,
    pathSegment: 'lovell-federal-health-care-va',
    variant: 'va',
  },
} as const

export const LOVELL_RESOURCE_TYPES = [
  RESOURCE_TYPES.STORY,
  RESOURCE_TYPES.STORY_LISTING,
  RESOURCE_TYPES.EVENT,
  RESOURCE_TYPES.EVENT_LISTING,
  RESOURCE_TYPES.PRESS_RELEASE,
  RESOURCE_TYPES.PRESS_RELEASE_LISTING,
  RESOURCE_TYPES.STAFF_PROFILE,
  RESOURCE_TYPES.VAMC_SYSTEM,
  RESOURCE_TYPES.LEADERSHIP_LISTING,
  RESOURCE_TYPES.LOCATIONS_LISTING,
  RESOURCE_TYPES.VAMC_HEALTH_SERVICES_LISTING,
]

export const LOVELL_BIFURCATED_RESOURCE_TYPES = [
  RESOURCE_TYPES.STORY,
  RESOURCE_TYPES.EVENT,
  RESOURCE_TYPES.STAFF_PROFILE,
  RESOURCE_TYPES.PRESS_RELEASE,
  RESOURCE_TYPES.LOCATIONS_LISTING,
  RESOURCE_TYPES.VAMC_HEALTH_SERVICES_LISTING,
]
