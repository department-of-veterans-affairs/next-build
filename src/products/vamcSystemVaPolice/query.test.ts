/**
 * @jest-environment node
 */

import { queries } from '@/data/queries'
import mockData from '@/products/vamcSystemVaPolice/mock.json'
import { RESOURCE_TYPES } from '@/lib/constants/resourceTypes' // Import RESOURCE_TYPES

jest.mock('@/lib/drupal/query', () => ({
  ...jest.requireActual('@/lib/drupal/query'),
  fetchSingleEntityOrPreview: () => mockData, // Mock fetchSingleEntityOrPreview
  getMenu: () => ({
    items: [],
    tree: [],
  }),
}))

describe('VamcSystemVaPolice formatData', () => {
  test('outputs formatted data', async () => {
    const result = await queries.getData(
      RESOURCE_TYPES.VAMC_SYSTEM_VA_POLICE, // Use the correct resource type
      { id: mockData.id }
    )
    expect(result).toMatchSnapshot()
  })
})
