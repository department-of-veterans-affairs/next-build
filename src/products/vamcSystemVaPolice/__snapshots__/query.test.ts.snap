// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VamcSystemVaPolice formatData outputs formatted data 1`] = `
{
  "breadcrumbs": [
    {
      "options": [],
      "title": "Home",
      "uri": "https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/",
    },
    {
      "options": [],
      "title": "VA Central Arkansas health care",
      "uri": "https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/central-arkansas-health-care",
    },
    {
      "options": [],
      "title": "About VA Central Arkansas",
      "uri": "internal:#",
    },
    {
      "options": [],
      "title": "VA police",
      "uri": "internal:#",
    },
  ],
  "entityId": 63857,
  "entityPath": "/central-arkansas-health-care/va-police",
  "faqs": {
    "displayAccordion": "1",
    "header": "Other questions you may have about VA police",
    "id": null,
    "intro": [],
    "questions": [
      {
        "answers": [
          {
            "html": "<p>VA police are armed and uniformed federal law enforcement officers. They provide law enforcement and security services to Veterans and their families at VA medical centers and other VA health facilities. Services include crime prevention and investigation, maintaining law and order, vehicle and foot patrols, telecommunication services, and workplace violence prevention.</p>",
            "id": "141565",
            "type": "paragraph--wysiwyg",
          },
        ],
        "id": "141566",
        "question": "What services do VA police provide?",
        "type": "paragraph--q_a",
      },
      {
        "answers": [
          {
            "html": "<p>VA police and local law enforcement work together to respond to crimes that happen on VA property. VA police and local law enforcement partners train together on skills and tactics such as active threat drills, crisis intervention training, disaster response, and more.</p>",
            "id": "141567",
            "type": "paragraph--wysiwyg",
          },
        ],
        "id": "141568",
        "question": "Do VA police work with local law enforcement?",
        "type": "paragraph--q_a",
      },
      {
        "answers": [
          {
            "html": "<p>90% of VA police have a military background. VA police is looking for qualified applicants to join its team. Candidates must have either criminal justice education, experience as a police officer, experience as a military police officer, or a combination of education and experience. Visit USAJOBS to search for current VA police job announcements.</p>

<p><a href="https://www.usajobs.gov/">Go to USAJOBS.gov to search for current VA police job announcements</a></p>

<p>Transitioning service members may be eligible for employment training, internship, and apprenticeship opportunities with VA police through the DOD SkillBridge program. For more information, email <a href="mailto:Skillbridge.vapolice@va.gov">Skillbridge.vapolice@va.gov</a>.</p>

<p><a href="https://skillbridge.osd.mil/">Learn more about the DOD SkillBridge program</a></p>",
            "id": "141569",
            "type": "paragraph--wysiwyg",
          },
        ],
        "id": "141570",
        "question": "Am I eligible to join the VA police force?",
        "type": "paragraph--q_a",
      },
      {
        "answers": [
          {
            "html": "<p>As a part of its mission to “protect those who served,” VA police across the country will begin to use in-car and body-worn cameras by the end of 2023. This policy increases the safety of VA facilities by promoting transparency and accountability.</p>",
            "id": "141571",
            "type": "paragraph--wysiwyg",
          },
        ],
        "id": "141572",
        "question": "Why do VA police wear body-worn cameras?",
        "type": "paragraph--q_a",
      },
      {
        "answers": [
          {
            "html": "<p>VA is committed to protecting the privacy of the Veterans, families, caregivers, and survivors we serve. All local VA privacy officers and VA police officers attend extensive privacy training before officers begin using body-worn cameras.</p>",
            "id": "141573",
            "type": "paragraph--wysiwyg",
          },
        ],
        "id": "141574",
        "question": "How will my privacy be protected?",
        "type": "paragraph--q_a",
      },
    ],
    "type": "paragraph--q_a_section",
  },
  "id": "7833b254-f4e4-4adb-bce1-d72d5e337c1e",
  "lastUpdated": "2023-11-20T02:56:16+00:00",
  "menu": {
    "data": null,
    "rootPath": "/central-arkansas-health-care/va-police/",
  },
  "metatags": [
    {
      "attributes": {
        "content": "VA police | VA Central Arkansas health care | Veterans Affairs",
        "name": "title",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "VA police officers help make VA medical centers and other VA health facilities safe for Veterans and their family members.",
        "name": "description",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "Veterans Affairs",
        "property": "og:site_name",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "VA police | VA Central Arkansas health care | Veterans Affairs",
        "property": "og:title",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "VA police officers help make VA medical centers and other VA health facilities safe for Veterans and their family members.",
        "property": "og:description",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "U.S. Department of Veterans Affairs",
        "property": "og:image:alt",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "summary_large_image",
        "name": "twitter:card",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "@DeptVetAffairs",
        "name": "twitter:site",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "VA police | Veterans Affairs",
        "name": "twitter:title",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "U.S. Department of Veterans Affairs",
        "name": "twitter:image:alt",
      },
      "tag": "meta",
    },
    {
      "attributes": {
        "content": "https://www.va.gov/img/design/logo/va-og-image.png",
        "name": "twitter:image",
      },
      "tag": "meta",
    },
  ],
  "moderationState": "published",
  "path": "/central-arkansas-health-care/va-police",
  "phoneNumber": {
    "extension": "",
    "id": "2bc3cc07-58a2-4d7f-b639-a90e0b890d7b",
    "number": "501-257-6550",
    "phoneType": "tel",
    "type": "paragraph--phone_number",
  },
  "policeOverview": {
    "html": "<p>VA police officers help make VA medical centers and other VA health facilities safe for Veterans and their family members.</p>
",
    "id": "141558",
    "type": "paragraph--wysiwyg",
  },
  "policeReport": {
    "description": "<p>To request a VA police report, you'll need to file a Freedom of Information Act (FOIA) request to your local VA privacy officer.</p>
",
    "id": "",
    "link": {
      "id": "",
      "label": "Learn how to file a FOIA request",
      "type": "paragraph--button",
      "url": "https://department.va.gov/foia/",
    },
    "title": "How to request a VA police report",
    "type": "paragraph--featured_content",
  },
  "published": true,
  "system": "VA Central Arkansas health care",
  "title": "VA police",
  "type": "node--vamc_system_va_police",
}
`;
